### DISCUSSION TITLE
Selection criteria for paying travel/accommodation

### DISCUSSION THREAD

**Discussion by The-Compiler** [pytest-dev/sprint#4] [General]
We've long mentioned that a good allocation of our [OpenCollective funds](https://opencollective.com/pytest) would be to get people to a sprint, and pay for their travel and accommodation.

I'd love to do that (see #1), but obviously that brings the trouble of *how* to select people. We currently have a balance of roughly $55k USD. That's considerably more funding than what we had for the sprint [in 2016](https://www.indiegogo.com/projects/python-testing-sprint-mid-2016)!

Now this brings a lot of questions I don't want to decide alone. For starters:

- How much of that should we budget for the sprint?
- How do we decide whose accommodation/travel we pay for, and who we sadly unable to pay inside of that budget?

For the latter, obviously travel from outside Europe will be far more costly, especially considering that the booking is relatively short notice (sorry...). @okken has expressed interest, and obviously I'd love to get him on board in person! Same for other regular contributors from outside Europe (@asottile if still relevant?).  For people who are newcomers especially, maybe we should focus more on people coming from Europe this time around, and then look at this again next year (in the hope we'll repeat this with a bit more planning time for booking travel)?

@pytest-dev/core I'd love your input on this. While I am one of the OpenCollective admins, obviously this is not something I want to decide on my own.

**Comment by Zac-HD** [comment]
Quick thoughts:
- 15-20k is a decent increase relative to the last sprint, but would still allow us to plan for several years of sprints.
  - I'd also be happy to support maintainers and core devs attending PyCon US or EuroPython (etc?) if they'd otherwise be unable to attend.
- If your employer can cover your costs (and you're not self-employed), please do that.
- Core devs, maintainers, notable contributors: generally happy to cover costs to let them attend.  If we'd reun out of budget, I'd prioritize a mix of that order and cost-per-person.
- Newcomers: no idea here, I'm a bit nervous from my experiences at PyCon sprints where I often end up finishing newcomers PRs after they leave.  Probably depends on the applications we get.

**Reply by flub** [reply]
i roughly agree with this i think

**Reply by The-Compiler** [reply]
FWIW we ended up having ~5 sign-ups, all from central Europe - so it looks like we can accept all, and stay well below the 15-20k budget. While a small group, it looks like we have a good split between core people and plugin authors / novices.

**Reply by nicoddemus** [reply]
Nice!

Thanks again to @The-Compiler for organizing the whole thing, and sorry not being able to help. Hopefully next time!

**Reply by The-Compiler** [reply]
Quick retrospective on this: We currently [paid out](https://opencollective.com/pytest/expenses) a total of around 3500€/$3700 of expenses related to the sprint.

**Reply by nicoddemus** [reply]
From my POV that's worth the money (and actually less than I antecipated). :+1:
